api/lib/application/certification-center-memberships/certification-center-membership-controller.js

```diff
@@ -3,18 +3,7 @@ import * as requestResponseUtils from '../../infrastructure/utils/request-respon
 import * as certificationCenterMembershipSerializer from '../../infrastructure/serializers/jsonapi/certification-center-membership-serializer.js';
 import { BadRequestError } from '../http-errors.js';
 
-const disable = async function (request, h, dependencies = { requestResponseUtils }) {
-  const certificationCenterMembershipId = request.params.id;
-  const pixAgentUserId = dependencies.requestResponseUtils.extractUserIdFromRequest(request);
-
-  await usecases.disableCertificationCenterMembership({
-    certificationCenterMembershipId,
-    updatedByUserId: pixAgentUserId,
-  });
-  return h.response().code(204);
-};
-
-const update = async function (
+const updateFromPixAdmin = async function (
```

Comment:
Pinaillage sur le message de commit : 

<img width="573" alt="Capture d’écran 2023-12-07 à 10 00 51" src="https://github.com/1024pix/pix/assets/5627688/751f5356-eb1e-4df3-9055-4467e860c474">

On aurait pu le rendre plus lisible en mettant uniquement l'essentiel dans l'en-tête et les informations complémentaires dans le corps :

> refactor(api): make function name more specific
>
> in anticipation of new function updateFromPixCertif 
> and reorder functions in controller

Et on aurait pu rendre sa lecture encore plus simple en faisant deux commits, un pour le renommage et un pour réordonner les commits (un "and" dans un message de commit est souvent le signe qu'on fait trop de choses dans un même commit)

Reply:
Nous avons fait deux commits et nous les avons renommés selon ta proposition.